apps/omg_watcher/priv/swagger/watcher_api_specs.yaml

```diff
@@ -0,0 +1,588 @@
+openapi: 3.0.0
+info:
+  version: '1.0.0'
+  title: Watcher security-critical API
+  description: >
+    API specification of the Watcher's security-critical Service
+    
+    Error codes are available in [html](https://github.com/omisego/elixir-omg/docs/api_specs/includes/errors.md#error-codes-description) format.
+  contact:
+    name: OmiseGO
+    email: omisego@omisego.co
+  license:
+    name: 'Apache 2.0: https://www.apache.org/licenses/LICENSE-2.0'
+    url: 'https://omisego.network/'
+
+tags:
+  - name: Status
+    description: Status of the child chain.
+  - name: UTXO
+    description: UTXO related API.
+  - name: Transaction
+    description: Transaction related API.
+  - name: InflightExit
+    description: InflightExit related API.    
+
+paths:
+  /status:
+    post:
```

Comment:
Where did you move definitions of `Byzantine events`?

Reply:
Events per se can't be documented in Swagger. I'll link corresponding api_specs file